index.js

```diff
@@ -58,6 +58,19 @@ client.on('ready', () => {
   client.user.setActivity('Praise be to ;;toromi')
 })
 
+// join a guild
+client.on('guildCreate', guild => {
+  const EXPECTED_GUILD_ID = process.env.GUILD_ID
+  console.log(`Guild: Joined ${guild.name}`)
+
+  if (EXPECTED_GUILD_ID) {
+    if (EXPECTED_GUILD_ID !== guild.id) {
```

Comment:
we can do `if (EXPECTED_GUILD_ID && EXPECTED_GUILD_ID !== guild.id)` lol

Reply:
it's more denpa this way